src/contact.rs

```diff
@@ -356,7 +356,9 @@ impl Contact {
 
                 if !name.as_ref().is_empty() {
                     if !row_name.is_empty() {
-                        if origin >= row_origin && name.as_ref() != row_name {
```

Comment:
would you mind adding doclines to tell what name, row_name and row_authname are here? would help me to follow the logic.

Reply:
i do

Reply:
wonderful, thanks!